src/shared/components/common/markdown-textarea.tsx

```diff
@@ -234,6 +241,7 @@ export class MarkdownTextArea extends Component<
                     "form-control border-0 rounded-top-0 rounded-bottom",
                     {
                       "d-none": this.state.previewMode,
+                      "is-invalid": contentLength > maxLength,
```

Comment:
We might want to put an actual `invalid` prop on the textarea, too.

Reply:
`textarea` only has `aria-invalid`, which I added now. Is this what you had in mind or is there some other way I can mark it as invalid?